subsystem/manifests_test.go

```diff
@@ -138,6 +139,44 @@ spec:
 		})
 	})
 
+	It("cluster doesn't exist", func() {
+		non_exiting_id := strfmt.UUID(uuid.New().String())
+		By("create manifest returns not found")
+		_, err := userBMClient.Manifests.V2CreateClusterManifest(ctx, &manifests.V2CreateClusterManifestParams{
+			ClusterID: non_exiting_id,
+			CreateManifestParams: &models.CreateManifestParams{
+				Content:  &base64Content,
+				FileName: &manifestFile.FileName,
+				Folder:   &manifestFile.Folder,
+			},
+		})
+		Expect(err).To(BeAssignableToTypeOf(installer.NewGetInfraEnvNotFound()))
```

Comment:
I don't understand this. Why would we return an infraEnv error for a cluster-related API?

Reply:
oh right, it should probably be NewV2GetClusterNotFound

Reply:
Yes. It was an copy paste mistake. Fixing